### PR TITLE
Test: Update test formatting

### PR DESCRIPTION
**Pull request by joeldenverdev**
- Update to format of tests we made as a group for the Cell class.  Each test contained in a unique `it` block.
- Please review for completeness of our original tests (especially the last one for sinking the ship)

### CODE REVIEW

File: spec/cell_spec.rb [hunk 1]
```diff
@@ -8,69 +8,107 @@
...
+  end
 
-    it 'fired_upon? returns boolean' do
+  describe '#fired_upon?' do
+    it 'returns false when the cell has been fired upon' do
       cell = Cell.new("B4")
-
       cruiser = Ship.new("Cruiser", 3)
       cell.place_ship(cruiser)
       expect(cell.fire_upon?).to be false
```
Comment (jaw772) [inline spec/cell_spec.rb:51]: needs to be fired_upon?

File: spec/cell_spec.rb [hunk 2]
```diff
@@ -8,69 +8,107 @@
...
-
       expect(cell_1.render).to eq(".")
       expect(cell_2.render).to eq(".")
       expect(cell_3.render).to eq(".")
+    end
 
+    it 'displays S in the cell when the optional argument is used' do
+      cell_1 = Cell.new("B4")
+      cell_2 = Cell.new("B2")
+      cell_3 = Cell.new("B3")
```
Comment (jaw772) [inline spec/cell_spec.rb:77]: in this block we don't use cell_2 and cell_3, so we should delete those

File: spec/cell_spec.rb [hunk 3]
```diff
@@ -8,69 +8,107 @@
...
+      cruiser = Ship.new("Cruiser", 3)
+      cell_1.place_ship(cruiser)
       expect(cell_1.render(true)).to eq("S")
+    end
 
+    it 'displays H when a ship has been hit in the cell' do
+      cell_1 = Cell.new("B4")
+      cell_2 = Cell.new("B2")
+      cell_3 = Cell.new("B3")
```
Comment (jaw772) [inline spec/cell_spec.rb:86]: in this block we don't use cell_2 and cell_3, so we should delete those

File: spec/cell_spec.rb [hunk 4]
```diff
@@ -8,69 +8,107 @@
...
+      cruiser = Ship.new("Cruiser", 3)
       cell_1.fire_upon
       expect(cell_1.fired_upon?).to be true
       expect(cell_1.render).to eq("H")
+    end
 
+    it 'displays M when a cell is fired upon but is a miss' do
+      cell_1 = Cell.new("B4")
+      cell_2 = Cell.new("B2")
+      cell_3 = Cell.new("B3")
```
Comment (jaw772) [inline spec/cell_spec.rb:96]: in this block we don't use cell_1 and cell_3, so we should delete those

File: spec/cell_spec.rb [hunk 5]
```diff
@@ -8,69 +8,107 @@
...
+      cruiser = Ship.new("Cruiser", 3)
       cell_2.fire_upon
       expect(cell_2.render).to eq("M")
+    end
 
+    it 'displays X when a ship has sunk' do
+      cell_1 = Cell.new("B4")
+      cell_2 = Cell.new("B2")
+      cell_3 = Cell.new("B3")
```
Comment (jaw772) [inline spec/cell_spec.rb:105]: in this block we don't use cell_2 and cell_3, so we should delete those

File: spec/cell_spec.rb (hunk 2 above)
Comment (kevingloss) [reply spec/cell_spec.rb:77]: Great points on having unnecessary variables.

File: spec/cell_spec.rb (hunk 4 above)
Comment (kevingloss) [reply spec/cell_spec.rb:96]: A good catch and we can update the naming of the variable so we don't start at cell_2.

**Pull request merged**